### PR TITLE
Victor dym

### PR DESCRIPTION
**Pull request by victor1dyminskyi**
Done HW 1, 2, 3

### CODE REVIEW

File: HW3/HW3.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+32.
+33.Although that way may not be obvious at first unless you're Dutch.
+34. Now is better than never.
+35. Although never is often better than *right* now.
+36. If the implementation is hard to explain, it's a bad idea.
+37. If the implementation is easy to explain, it may be a good idea.
+38. Namespaces are one honking great idea -- let's do more of those!Beautiful is better than ugly.
+39.Explicit is better than implicit.
+"""
+
```
Comment (kolyasalubov) [inline HW3/HW3.py:44]: good job

File: HW4/HW4.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+c = int(input("Enter the temperature in Celsius: "))
```
Comment (kolyasalubov) [inline HW4/HW4.py:1]: good job

File: HW5/HW5.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# 1 TASK---------------------------------------------
+
```
Comment (kolyasalubov) [inline HW5/HW5.py:2]: look at this implementation

list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)

File: HW5/HW5.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+#     if i < 10:
+#         fib = li1[i] + li1[i+1]
+#         add = li1.append(fib)
+#         print(i, i+1, li1[i], li1[i + 1], li1)
+#     else:
+#         print("done")
+
+# print(li1)
+
+
```
Comment (kolyasalubov) [inline HW5/HW5.py:28]: good job

File: HW5/HW5.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+#
+# if factorial >= 0:
+#     for i in range(factorial):
+#         fact = list[-1] * (i+1)
+#         # print(fact)
+#         list.append(fact)
+#     print(f"your result is {list[-1]}")
+# else:
+#     print("error, your number {} cannot be used".format(factorial))
+
```
Comment (kolyasalubov) [inline HW5/HW5.py:55]: look at this

input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

File: HW5/HW5.py (hunk 3 above)
Comment (victor1dyminskyi) [reply HW5/HW5.py:2]: Ok, I`ll check

File: HW5/HW5.py (hunk 5 above)
Comment (victor1dyminskyi) [reply HW5/HW5.py:55]: Ok, I'll check

File: HW6/HW6.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# ---------------------------------------------------------------------------
+# 1 TASK
+
+list_even = []
```
Comment (kolyasalubov) [inline HW6/HW6.py:4]: good job

File: HW6/HW6.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    else:
+        list_else.append(i)
+
+print("Here is the list of even numbers that are divisible by 2 ", list_even)
+print("Here is the list of odd numbers that are divisible by 3: {}".format(list_odd))
+print(f"Here is the list of all left numbers: {list_else}")
+
+# ---------------------------------------------------------------------------
+# 2 TASK
+
```
Comment (kolyasalubov) [inline HW6/HW6.py:21]: look at this implementation

def check_login():
    login = "First"
    while True:
        input_login = input("Enter your login: ")
        if input_login == login:
            print("Nice, you passed ")
            break
        else:
            print("Try again ")

File: HW7/HW7.py [hunk 8]
```diff
@@ -0,0 +1,70 @@
+# -------------------------------------------------------
```
Comment (kolyasalubov) [inline HW7/HW7.py:1]: good job

**Pull request merged**